emperor/format.py

```diff
@@ -553,7 +553,8 @@ def format_emperor_autograph(metadata_fp, coords_fp, language='HTML'):
 _BIPLOT_VISIBILITY_SELECTOR = """
             <br>
             <form name="biplotsvisibility">
-            <input type="checkbox" onClick="toggleBiplotVisibility()" checked>Biplots Visibility</input>
+            <input type="checkbox" onClick="toggleBiplotVisibility(false)" checked>Taxa Sphere Visibility</input>
```

Comment:
Can you add a `<br>` tag in-between these lines? this will make the presentation nicer:

![screen shot 2015-03-04 at 2 51 44 pm](https://cloud.githubusercontent.com/assets/375307/6495365/32b1be18-c27e-11e4-93c2-cd50076564ab.png)